### PR TITLE
fix: allow passing WebContents to the window option

### LINKED ISSUE #184: v4.0.2+ breaks if `window` option is `WebContents`
This PR https://github.com/sindresorhus/electron-context-menu/pull/182 breaks code if `window` option is not `BrowserWindow`.

On this line https://github.com/lovemegowin/electron-context-menu/blob/79bf5666d84fc0fa05cfbb658160c2e5db5f5263/index.js#L382, `win` is not always `BrowserWindow`, it could be `WebContents`: https://github.com/sindresorhus/electron-context-menu/blob/691c307dcedd18f4c02dd385e98610c3a8d20dd1/index.d.ts#L138

For example, this following code doesn't work with v4.0.2+

```js
const browserView = new BrowserView();
contextMenu({ window: browserView.webContents });
```

CC: @lovemegowin @sindresorhus

Btw, in the future, for behavior changes like this, we should follow `semver`.

### PR DESCRIPTION
**Pull request by quanglam2807**
Resolve #184

CC: @lovemegowin @sindresorhus

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -377,9 +377,7 @@ export default function contextMenu(options = {}) {
 			disposeMenu();
 		};
 
-		if (win.once !== undefined) { // Support for BrowserView
-			win.webContents.once('destroyed', disposable);
-		}
+		webContents(win).on('destroyed', disposable);
```
Comment (quanglam2807) [inline index.js:380]: `win` is not always `BrowserWindow`, so we must use utils to get `WebContents`: https://github.com/sindresorhus/electron-context-menu/blob/691c307dcedd18f4c02dd385e98610c3a8d20dd1/index.js#L7

**Comment by sindresorhus** [comment]
Ensure you have manually tested this with the supported typed: `BrowserWindow | BrowserView | WebViewTag | WebContents`

File: index.js [hunk 2]
```diff
@@ -377,9 +377,7 @@ export default function contextMenu(options = {}) {
 			disposeMenu();
 		};
 
-		if (win.once !== undefined) { // Support for BrowserView
-			win.webContents.once('destroyed', disposable);
-		}
+		webContents(win).once('destroyed', disposable);
```
Comment (quanglam2807) [inline index.js:380]: `win` is not always `BrowserWindow`, so we must use utils to get `WebContents`: https://github.com/sindresorhus/electron-context-menu/blob/691c307dcedd18f4c02dd385e98610c3a8d20dd1/index.js#L7

**Comment by quanglam2807** [comment]
@sindresorhus Thanks for reminding! I've tested it.

**Pull request merged**